### PR TITLE
system-variables: explain tidb_enable_concurrent_ddl

### PR DESCRIPTION
**Pull request by wjhuang2016**
Signed-off-by: wjhuang2016 <huangwenjun1997@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -738,6 +738,17 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`OFF`
 - 这个变量用于控制是否开启 TiDB 对 `PREDICATE COLUMNS` 的收集。关闭该变量后，之前收集的 `PREDICATE COLUMNS` 会被清除。详情见[收集部分列的统计信息](/statistics.md#收集部分列的统计信息)。
 
+### `tidb_enable_concurrent_ddl` <span class="version-mark">从 v6.2.0 版本开始引入</span>
+
+> **警告：**
+>
+> 禁止用户私自修改该变量。
```
Comment (TomShawn) [inline system-variables.md:745]: 可以再说下后果吗？
```suggestion
> 禁止修改该变量。
```

File: system-variables.md [hunk 2]
```diff
@@ -738,6 +738,17 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 默认值：`ON`
+- 这个变量用于控制是否开启 TiDB 使用并发 DDL。
```
Comment (TomShawn) [inline system-variables.md:750]: 可以稍加解释什么是并发 DDL 吗？
```suggestion
- 这个变量用于控制 TiDB 是否使用并发 DDL 语句。
```

File: system-variables.md [hunk 3]
```diff
@@ -738,6 +738,17 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`OFF`
 - 这个变量用于控制是否开启 TiDB 对 `PREDICATE COLUMNS` 的收集。关闭该变量后，之前收集的 `PREDICATE COLUMNS` 会被清除。详情见[收集部分列的统计信息](/statistics.md#收集部分列的统计信息)。
 
+### `tidb_enable_concurrent_ddl` <span class="version-mark">从 v6.2.0 版本开始引入</span>
+
+> **警告：**
+>
+> 禁止用户修改该变量，该变量后续将废除，并且开启后风险不确定，有可能导致集群元数据出错。
```
Comment (TomShawn) [inline system-variables.md:745]: 已经是默认开启了，是说关闭吗？
```suggestion
> 禁止用户修改该变量，因为关闭后风险不确定，有可能导致集群元数据出错。该变量计划于后续 TiDB 版本中废弃。
```

File: system-variables.md [hunk 4]
```diff
@@ -738,6 +738,17 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+### `tidb_enable_concurrent_ddl` <span class="version-mark">从 v6.2.0 版本开始引入</span>
+
+> **警告：**
+>
+> 禁止用户修改该变量，因为关闭后风险不确定，有可能导致集群元数据出错。该变量计划于后续 TiDB 版本中废弃。
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 默认值：`ON`
+- 这个变量用于控制是否让 TiDB 使用并发 DDL。在开启并发 DDL 后，DDL 执行流程改变，DDL 不容易被其他 DDL 阻塞，并且能够同时添加多个索引。
```
Comment (TomShawn) [inline system-variables.md:750]:
```suggestion
- 这个变量用于控制是否让 TiDB 使用并发 DDL 语句。在开启并发 DDL 语句后，DDL 语句的执行流程有所改变，DDL 语句不容易被其他 DDL 语句阻塞，并且能够同时添加多个索引。 
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by easonn7** [comment]
LGTM，这个是内部保护开关，默认开，不建议放开到官网文档，预定 7.0 后废弃该参数。

File: system-variables.md [hunk 5]
```diff
@@ -738,6 +738,17 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`OFF`
 - 这个变量用于控制是否开启 TiDB 对 `PREDICATE COLUMNS` 的收集。关闭该变量后，之前收集的 `PREDICATE COLUMNS` 会被清除。详情见[收集部分列的统计信息](/statistics.md#收集部分列的统计信息)。
 
+### `tidb_enable_concurrent_ddl` <span class="version-mark">从 v6.2.0 版本开始引入</span>
+
+> **警告：**
+>
+> 禁止用户修改该变量，因为关闭后风险不确定，有可能导致集群元数据出错。该变量计划于后续 TiDB 版本中废弃。
```
Comment (TomShawn) [inline system-variables.md:745]:
```suggestion
> **请勿修改该变量值**，因为关闭后风险不确定，有可能导致集群元数据出错。该变量计划于后续 TiDB 版本中废弃。
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: system-variables.md [hunk 6]
```diff
@@ -738,6 +738,17 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`OFF`
 - 这个变量用于控制是否开启 TiDB 对 `PREDICATE COLUMNS` 的收集。关闭该变量后，之前收集的 `PREDICATE COLUMNS` 会被清除。详情见[收集部分列的统计信息](/statistics.md#收集部分列的统计信息)。
 
+### `tidb_enable_concurrent_ddl` <span class="version-mark">从 v6.2.0 版本开始引入</span>
+
+> **警告：**
+>
+> **请勿修改该变量值**，因为关闭后风险不确定，有可能导致集群元数据出错。该变量计划于后续 TiDB 版本中废弃。
```
Comment (TomShawn) [inline system-variables.md:745]:
```suggestion
> **请勿修改该变量值**，因为关闭后风险不确定，有可能导致集群元数据出错。
```

**Pull request merged**